Opcodes/CMakeLists.txt

```diff
@@ -23,6 +23,9 @@ message(STATUS "## Building Plugin Opcodes ##")
 make_plugin(doppler doppler.cpp)
 make_plugin(fractalnoise tl/fractalnoise.cpp)
 make_plugin(ftsamplebank ftsamplebank.cpp)
+find_path(DIRENT_INCLUDE_DIR "dirent.h" REQUIRED)
+target_include_directories(ftsamplebank PRIVATE ${DIRENT_INCLUDE_DIR})
```

Comment:
Is this for all platforms or just MSVC? There is a VCPKG for dirent on Windows. For linux, it should be a system file no?

Reply:
Sure, either way the header is required though